### ISSUE TITLE
Trying to use WebSockets as SignalR transport via WWW Proxy

### ISSUE THREAD

**Issue by Arkasian** [dotnet/core#4583]
# Issue Title
Trying to use WebSockets as SignalR transport via WWW Proxy

# General
Technical info at the beginning:
**OS**: Debian 10 "buster" (upgrade from Debian 9, kernel 4.19)
**SDK**: .NET Core SDK 3.1.100 is used to build.
**Runtime**: .NET Core Runtime 3.1.1 is used to deploy an app.
**WWW Proxy**: Apache 2.4.25
**SignalR** of version 2.4.1.

So my issue is: I want to use WS as a transport for my website, but whenever I will try to connect using it, SignalR throws the following error on client-side:
``
Firefox can’t establish a connection to the server at wss://dev-core.lukas-bownik.net/hubs/media?id=zbGWDjVqkGEOW47lSeWwqQ. signalr.min.js:16:109830
``
``
[2020-04-17T15:49:24.647Z] Error: Failed to start the transport 'WebSockets': Error: There was an error with the transport.
``
If you want to see it for yourself: [url to my dev instance of app](https://dev-core.lukas-bownik.net/Storage/Browse?path=%2FWallpapers_Gifs)
And here is my configuration for SignalR:
[Startup.cs](https://github.com/Arkasian/pika-core/blob/development/Startup.cs)
And here is my configuration for Kestrel for this app:
[Program.cs](https://github.com/Arkasian/pika-core/blob/development/Program.cs)
I think that there is something wrong, but I just can't see it, or is there any error inside Core or SR which makes it immpossible?
Correct me please, if I am mistaken.
Greetings!

**Comment by BrennanConroy** [comment]
> **SignalR** of version 2.4.1.

Your client is using ASP.NET SignalR which is not compatible with ASP.NET Core SignalR.
https://docs.microsoft.com/aspnet/core/signalr/version-differences?view=aspnetcore-3.1

**Comment by Arkasian** [comment]
No, it is ASP.NET Core SignalR, just a shortcut. I know it is not compatible, if it is not compatible one it won't even work for SSE or LP and for those two it works perfectly fine.

**Comment by BrennanConroy** [comment]
Well 2.4.1 is **not** a version of ASP.NET Core SignalR which is why your issue was confusing.

Moving on assuming you're using ASP.NET Core SignalR on both the client and server. Have you enabled websockets in your proxy?
